### DISCUSSION TITLE
Is it possible to communicate with the EMS-ESP over UART-USB?

### DISCUSSION THREAD

**Discussion by kami83** [emsesp/EMS-ESP32#1837] [General]
Hi, 

i would like to get the informations about my boiler directly from the USB-UART of my WEMOS S2 MINI? Is this possible? Is there a serial output? Thx a lot.

BR kami

**Comment by proddy** [comment]
No, you need a circuit to translate the 2-line EMS bus to serial TTL. See https://bbqkees-electronics.nl/wiki/gateway/introduction.html

**Reply by kami83** [reply]
Hi, sorry this was a little bit missunderstanding. I am using the EMS-ESP Bus with an ESP32 Wemos S2 Mini and the USB Port. I would like to get in contact with the ESP32 over Serial interface and see the data from the EMS-BUS like in the Webinterface or the API.

BR kami

**Reply by proddy** [reply]
You can use the telnet console to see the data. Explanation is in the doc

**Reply by kami83** [reply]
Okay. thx a lot.
